data-es6.js

```diff
@@ -16883,8 +16883,9 @@ exports.tests = [
         closure: true,
         typescript1corejs2: true,
         firefox2: true,
-        safari13: false,
-        safari15: false,
+        safari11: false,
+        safari15_6: false,
+        safari16_2: true,
```

Comment:
```suggestion
        safari16_2: false,
```

Reply:
Note that it's also possible the test is wrong, since it's not in strict mode?

Reply:
I think the test is fine: although it's not in strict mode, the array pattern `[id, id]` disqualifies simple parameter anyway. So duplicate bound names check should still kick in.

Reply:
right, but the syntax error in 16.2 i get is that you can't have a non-simple parameter list in sloppy mode at all.

Reply:
In other destructuring tests, e.g.

https://github.com/kangax/compat-table/blob/2c4d894c9a9aa48d6f23f186d884c8d23e307e13/data-es6.js#L16734C1-L16742

the test functions don't have strict directive either. Does Safari 16.2 throw syntax error on them as well? Or does it only throw SyntaxError on that specific case?

Reply:
That code returns true, so I'm not sure what the difference is.

Reply:
ok so it definitely doesn't have to do with strict mode - `var d = function d([e]) { return d };` works fine while `var d = function d([d]) { return d };` throws a syntax error. I think the issue is that this test is just wrong, because the var d declaration needs to be inside the `try`.

Reply:
> var d = function d([e]) { return d }; works fine while var d = function d([d]) { return d }; throws a syntax error.

Yeah that's exactly the Webkit bug behaviour. I have updated the test, can you take a look if fails (expectedly) on Safari 16.2?